### PR TITLE
magento/devdocs#: Add setPaymentMethodAndPlaceOrder mutation API errors

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds information about the API errors of [setPaymentMethodAndPlaceOrder](https://devdocs.magento.com/guides/v2.3/graphql/mutations/set-payment-place-order.html) mutation.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/graphql/mutations/set-payment-place-order.html

## Links to Magento source code

-  [Could not find a cart with ID "XXX"](https://github.com/magento/magento2/blob/2.3.5/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetPaymentMethodAndPlaceOrderTest.php#L211)
-  [Guest email for cart is missing.](https://github.com/magento/magento2/blob/2.3.5/app/code/Magento/QuoteGraphQl/Model/Resolver/SetPaymentAndPlaceOrder.php#L89)
-  [Required parameter "cart_id" is missing](https://github.com/magento/magento2/blob/2.3.5/app/code/Magento/QuoteGraphQl/Model/Resolver/SetPaymentAndPlaceOrder.php#L75)
-  [Required parameter "code" for "payment_method" is missing.](https://github.com/magento/magento2/blob/2.3.5/app/code/Magento/QuoteGraphQl/Model/Resolver/SetPaymentAndPlaceOrder.php#L80)
-  [The current user cannot perform operations on cart "XXX"](https://github.com/magento/magento2/blob/2.3.5/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetPaymentMethodAndPlaceOrderTest.php#L237)
-  [The shipping address is missing. Set the address and try again.](https://github.com/magento/magento2/blob/2.3.5/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetPaymentMethodAndPlaceOrderTest.php#L152)
-  [The requested Payment Method is not available.](https://github.com/magento/magento2/blob/2.3.5/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetPaymentMethodAndPlaceOrderTest.php#L196)
-  [Unable to place order: Some of the products are out of stock.](https://github.com/magento/magento2/blob/2.3.5/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetPaymentMethodAndPlaceOrderTest.php#L140)

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/graphql/mutations/set-payment-place-order.md [hunk 1]
```diff
@@ -82,3 +82,16 @@ The `placeOrderOutput` object contains the `order` object, which contains the fo
...
+`Guest email for cart is missing.` |  The guest attempted to place an order but did not provide an email address. See [setGuestEmailOnCart]({{ page.baseurl }}/graphql/mutations/set-guest-email.html) mutation.
+`Required parameter "cart_id" is missing` | The required `cart_id` argument contains an empty value.
+`Required parameter "code" for "payment_method" is missing.` | The value specified in the `code` argument is empty.
+`The current user cannot perform operations on cart "XXX"` | An unauthorized user (guest) tried to set a payment method and place an order with a customer's cart, or an authorized user (customer) tried to set a payment method and place an order with a cart of another customer.
+`The shipping address is missing. Set the address and try again.` | You ran `setPaymentMethodAndPlaceOrder` mutation before [setShippingAddressesOnCart]({{ page.baseurl }}/graphql/mutations/set-shipping-method.html). Set a shipping address first. [GraphQL checkout tutorial](https://devdocs.magento.com/guides/v2.3/graphql/tutorials/checkout/index.html) shows the order placement sequence.
```
Comment (dmytro-ch) [inline src/guides/v2.3/graphql/mutations/set-payment-place-order.md:95]:
```suggestion
`The shipping address is missing. Set the address and try again.` | You run `setPaymentMethodAndPlaceOrder` mutation before [setShippingAddressesOnCart]({{ page.baseurl }}/graphql/mutations/set-shipping-method.html). Set a shipping address first. [GraphQL checkout tutorial](https://devdocs.magento.com/guides/v2.3/graphql/tutorials/checkout/index.html) shows the order placement sequence.
```

File: src/guides/v2.3/graphql/mutations/set-payment-place-order.md [hunk 2]
```diff
@@ -82,3 +82,16 @@ The `placeOrderOutput` object contains the `order` object, which contains the fo
...
+`The requested Payment Method is not available.` | Specified in the `payment_method` argument payment method is disabled or does not exist.
```
Comment (dmytro-ch) [inline src/guides/v2.3/graphql/mutations/set-payment-place-order.md:96]:
```suggestion
`The requested Payment Method is not available.` | The payment method specified in the `payment_method` argument is disabled or does not exist.
```

**Comment by atwixfirster** [comment]
> Hi @atwixfirster,
> could you please apply a minor improvement?
> 
> Thank you!

**done**

Could you please review?

Thanks, @dmytro-ch !

File: src/guides/v2.3/graphql/mutations/set-payment-place-order.md (hunk 2 above)
Comment (atwixfirster) [reply src/guides/v2.3/graphql/mutations/set-payment-place-order.md]: **done**

**Comment by keharper** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**